guessing_game_v2.html

```diff
@@ -31,92 +30,68 @@
     }
     oneOrZero();
```

Comment:
Where is the oneOrZero() function that you're calling? Am I missing it? If not, remove this line of code since it's calling for an unknown function.